fastly/block_fastly_service_director.go

```diff
@@ -10,10 +10,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+// DirectorServiceAttributeHandler implements ServiceAttributeDefinition.
```

Comment:
My linter was complaining about the lack of doc strings, so I've added them where required.